source/fe/fe_series_fourier.cc

```diff
@@ -181,6 +181,20 @@ namespace FESeries
 
 
 
+  template <int dim, int spacedim>
+  void
+  Fourier<dim, spacedim>::precalculate_all_transformation_matrices()
+  {
+    for (unsigned int fe = 0; fe < fe_collection->size(); ++fe)
+      ensure_existence(*fe_collection,
```

Comment:
I think you could neatly use the following idiom here:
```
  Threads::TaskGroup<> task_group;
   for (unsigned int fe = 0; fe < fe_collection->size(); ++fe)
      task_group += Threads::new_task ([]() {ensure_existence(*fe_collection, ...}

  task_group.join_all();
```
to compute all of these things in parallel -- assuming that the `ensure_existence` serializes access to all member variables appropriately.

Doesn't have to happen for this patch, but keep it in mind :-)